src/style.css

```diff
@@ -0,0 +1,153 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul,
+p,
+h3 {
+  margin: 0;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+}
+
+p {
+  display: block;
+}
+
+a {
+  text-decoration: none;
+  display: block;
+}
+
+@font-face {
+  font-family: "roboto";
+  src: url("./fonts/Roboto-Regular.ttf") format("truetype");
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "roboto";
+  src: url("./fonts/Roboto-Medium.ttf") format("truetype");
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "roboto";
+  src: url("./fonts/Roboto-Bold.ttf") format("truetype");
+  font-weight: 700;
+  font-style: normal;
+}
+
+:root {
+  --primary-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --button-color: #00acdc;
+}
+
+html {
+  font-family: "roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  object-fit: contain;
+  object-position: center;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__name {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 150%;
+  color: var(--primary-text-color);
+}
+
+.card__product-code {
+  margin-bottom: 17px;
+  font-size: 10px;
+  color: var(--secondary-text-color);
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url("./images/star.svg") center no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 4) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
use all code for stars from prev task. If I change `stars--0` to `stars--5` in classes I want to see 5 active stars

Reply:
not fixed